src/ol_infrastructure/applications/airbyte/airbyte_server_policy.hcl

```diff
@@ -0,0 +1,23 @@
+path "secret-operations/global/odl_wildcard_cert" {
+  capabilities = ["read"]
+}
+
+path "postgres-airbyte/creds/app" {
+  capabilities = ["read"]
+}
+
+path "postgres-airbyte/creds/admin" {
```

Comment:
Yeah, I'm honestly not entirely clear on it myself but it has something to do with the 'temporal' service that runs along side airbyte. It wants to create a database(s). I didn't dig too deep into it but I suspect we can either bootstrap what it wants prior to starting airbyte the first time or we can adjust the approle to give it the specific powers it wants and nothing more. 

The same 'temporal' required turning off the 'forcessl' setting. It doesn't use the DATABASE_CONNECTION_STRING (or whatever the env var is called), so it doesn't get the two required parameters and refuses to speak with the DB because of it.